### PR TITLE
Ruby on Rails capstone project: Budget app

### PR DESCRIPTION
**Pull request by mohisa302**
### Budget app
In this PR I:

- **Splash screen**

  - [x] A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
  - [x] Sign up and log in pages
  - [x] The user is able to register in the app with full name, email and password (all mandatory).
  - [x] The user can log into the app using email and password.
  
-  **Home page (categories page)**
    - [x] When the user logs in, they are presented with the categories page.
    - [x] For each category, the user can see their name, icon and the total amount of all the expenses that belongs to that category.
    - [x] When the user clicks (or taps) on a category item, the application navigates to the expenses page for that category.
    - [x] "add a new category" to create a new category.

-  **Expense page**
    - [x] For a given category, the list of expenses is presented, ordered by the most recent.
    - [x] sum of all of the amounts of the expenses in that category.
    - [x] "add a new expense"  to create a new expense.
    - [x]  "Back" button (<)
-  **Added test and deployed**
- [Link to the loom video](https://www.loom.com/share/9e1d1d7f187648d7a6e9aa57f066f212)
- Email confirmation URL: `http://localhost:3000/letter_opener`

### CODE REVIEW

File: app/views/splash/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-    <button class=" focus:outline-none">sign in</button>
-  </a>
-</div>
+<div class="flex background flex-col items-center bg-blue-400 sm:text-center h-screen justify-center">
+  <h1 class="text-white splash-text my-2">Welcome to costino!</h1>
+
+  <a href="<%= new_user_session_path() %>" class="w-screen flex justify-center my-4 bg-white hover:bg-blue-500 text-blue-400 font-semibold hover:text-white py-2 px-4 border rounded">
+    <button class=" focus:outline-none">sign in</button>
+  </a>
+</div>
```
Comment (AnselemOdims) [inline app/views/splash/index.html.erb]: - [ ] Good job working on the splash screen , but it doesn't quite match the original design. Please kindly ensure it has both the `Sign Up` and `Login` buttons.

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @mohisa302 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/splash/index.html.erb (hunk 1 above)
Comment (mohisa302) [reply app/views/splash/index.html.erb]: Thanks i will implement the changes

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**